Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            m "A quite long [_round!t], [player]."
+
+            if len(store.mas_nou.game.monika.hand) < 4:
+                if store.mas_nou.player_win_streak > 0:
+                    m "And I almost win this time!"
+
+                else:
+                    m "And I almost win!"
+
+                m "Ahaha~ Well played!"
+
+            else:
+                m "Well played!"
+
+        elif choice == 1:
+            m "That was intense!"
+
+        else:
+            m "Ehehe~ You're really good!"
+
+    else:
+        $ choice = renpy.random.randint(0, 3)
+
+        if not choice:
+            if store.mas_nou.player_win_streak > 0:
+                m "You won again!"
+
+            else:
+                m "You won!~"
+
+        elif choice == 1:
+            m "This [_round!t] is yours!"
+
+        elif choice == 2:
+            m "And you won! Good job!"
+            if renpy.random.randint(0, 8) > 7:
+                m "But don't expect to win everytime~"
+
+        else:
+            if store.mas_nou.monika_win_streak > 1:
+                m "I'm glad you won this time~"
+
+            m "Good job, [player]!"
+    return
+
+label mas_nou_reaction_player_wins_game:
+    $ choice = renpy.random.randint(0, 3)
+
+    if not choice:
+        m "Oh!{w=0.2} Actually you won this game!"
+        # TODO: check here for saying NoU and total turns
+        m "I didn't notice you were so close to the victory."
+        m "Good job, ehehe~"
+
+    elif choice == 1:
+        m "Oh, and you won this game too!"
+        m "Congratulations! Ehehe~"
+
+    elif choice == 2:
+        m "Let's see.{w=0.2}.{w=0.2}.{w=0.2}{nw}"
+        m "Oh, [player]! You won this game!"
+        if mas_isMoniEnamored(higher=True) and not renpy.random.randint(0, 4):
+            # m "I'd give you a big boop!"
+            m "I would give you a big hug if I were near you~"
+            m "Ehehe~"
+        else:
+            m "That was fun!"
+
+    else:
+        m "...And you're the first who reached [persistent._mas_game_nou_house_rules[win_points]] points!"
+        m "Congrats, [player]~"
+    return
+
+label mas_nou_reaction_monika_wins_round:
+    if persistent._mas_game_nou_wins["Abandoned"] > 2:
+        m "I win!~"
+        m "Thanks for playing with me, [player], {w=0.2}{nw}"
+        extend "I'm sure you'll win next time!"
+
+    elif store.mas_nou.player_win_streak > 5:
+        $ choice = renpy.random.randint(0, 2)
+
+        if not choice:
+            if len(store.mas_nou.game.player.hand) > 4:
+                m "I won!"
+                m "..."
+                m "Not without your help, I guess. Ehehe~"
+
+            else:
+                m "Told you I'll win!"
+                m "Now it's time for you to draw cards."
+
+        elif choice == 1:
+            m "Ahaha! My luck is back~"
+
+        else:
+            m "There we go!"
+            m "I finally won you~"
```

Comment:
Unless I misunderstood the context. It should be "I finally won~". "I finally won you~" sounds like she won the player as a prize.

Reply:
~~But puns~~Fixed